### ISSUE TITLE
Non-backwards compatible change in moving_averages.assign_moving_average

### ISSUE THREAD

**Issue by fchollet** [tensorflow/tensorflow#6270]
A `zero_debias` argument has recently been added to `moving_averages.assign_moving_average`. `zero_debias=True` significantly changes the behavior of `moving_averages.assign_moving_average`, in an unexpected way.

The default value for `zero_debias` is True. It should be False. The True default breaks existing code (it was breaking Keras), and it also a bad default value on its own because it introduces unexpected behavior (one would expect `assign_moving_average` to merely assign a moving average, i.e. return the equivalent of an `assign_add` op. Instead it creates weights, etc).

**Comment by ebrevdo** [comment]
It's good to discuss this before 1.0 release.  Most people do want debiasing.  We should fix keras to use this function properly or support debiasing as well.

**Comment by ebrevdo** [comment]
(we should mention this as a breaking change in RELEASE.md)

**Comment by ppwwyyxx** [comment]
Is there any update on this since 1.0 is already in alpha now.

**Comment by joel-shor** [comment]
Sorry, I'm not sure what the action item is. Keras, in particular, was fixed internally on Dec 14, 2016 and was included in the next rollout. If that was the action item, then this can be closed.

If instead, the action item was to change the default of `zero_debias` in `assign_moving_average` to `False`, then I think more discussion is required, especially since it apparently improves some of the standard implementations of batchnorm, which in turn has statistically significant quality improvements on things like benchmark inception.

I will close this thread, assuming it is the first. If it is not, please reopen, assign to me, and perhaps give more details as to why the default should be moved to `false`.
